inc/saihostintf.h

```diff
@@ -230,6 +230,18 @@ typedef enum _sai_hostif_trap_id_t
     /** default packet action is forward */
     SAI_HOSTIF_TRAP_ID_MLD_V2_REPORT = 0x0000200f,
 
+    /** IP packets to local router IP address (routes with SAI_ROUTE_ATTR_NEXT_HOP_ID=SAI_SWITCH_ATTR_CPU_PORT)
+     * (default packet action is trap) */
+    SAI_HOSTIF_TRAP_ID_IP2ME = 0x00002010,
+
+    /** SSH traffic to local router IP address 
```

Comment:
Ports info added

From: lguohan [mailto:notifications@github.com]
Sent: Tuesday, May 17, 2016 6:37 PM
To: opencomputeproject/SAI <SAI@noreply.github.com>
Cc: Itai Baz <itaib@mellanox.com>; Author <author@noreply.github.com>
Subject: Re: [opencomputeproject/SAI] Add new traps : IP2ME, SSH and SNMP (#105)


In inc/saihostintf.h<https://github.com/opencomputeproject/SAI/pull/105#discussion_r63546319>:

> @@ -230,6 +230,18 @@ typedef enum _sai_hostif_trap_id_t

>      /** default packet action is forward */

>      SAI_HOSTIF_TRAP_ID_MLD_V2_REPORT = 0x0000200f,

>

> +    /** IP packets to local router IP address (routes with SAI_ROUTE_ATTR_NEXT_HOP_ID=SAI_SWITCH_ATTR_CPU_PORT)

> +     * (default packet action is trap) */

> +    SAI_HOSTIF_TRAP_ID_IP2ME = 0x00002010,

> +

> +    /** SSH traffic to local router IP address

can you add dest port = 22 criteria to the comment to make this trap more clear?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/opencomputeproject/SAI/pull/105/files/f346e1faf895e7c2127d66df5b0dc0a24229a843#r63546319>